pallets/phala/src/stakepool.rs

```diff
@@ -437,16 +433,35 @@ pub mod pallet {
 					payout_commission: None,
 					owner_reward: Zero::zero(),
 					cap: None,
-					reward_acc: CodecFixedPoint::zero(),
 					total_shares: Zero::zero(),
 					total_stake: Zero::zero(),
 					free_stake: Zero::zero(),
-					releasing_stake: Zero::zero(),
 					workers: vec![],
+					cd_workers: vec![],
 					withdraw_queue: VecDeque::new(),
 				},
 			);
 			PoolCount::<T>::put(pid + 1);
+			// TODO(mingxuan): create_collection should return cid
+			let collection_id: CollectionId = pallet_rmrk_core::Pallet::<T>::collection_index();
+			#[cfg(not(feature = "std"))]
+			use alloc::format;
+			#[cfg(feature = "std")]
+			use std::format;
```

Comment:
Don't mix `use` and logic code. Put it in the beginning of the function, or better put it at the beginning of `mod`